### ISSUE TITLE
An error in "develop" branch

### ISSUE THREAD

**Issue by albertly** [bjerkio/oidc-react#173]
I've packed "develop" branch with command "npm pack" to make tgz file.
(I need to make some changes in oidc-react)
When using oidc-react as packed tgz in the project "example-oidc-react" (npm install ../oidc-react/oidc-react-1.0.0-alpha.5.tgz)
I've got an error when running in a browser: 
--------------------------------------------------------------------------------------
Uncaught TypeError: this._signinStart is not a function
    at Object.t [as signIn] (oidc-client.min.js:47)
    at onClick (LoggedIn.tsx:12)
    at HTMLUnknownElement.callCallback (react-dom.development.js:188)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:237)
    at invokeGuardedCallback (react-dom.development.js:292)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:306)
    at executeDispatch (react-dom.development.js:389)
    at executeDispatchesInOrder (react-dom.development.js:414)
    at executeDispatchesAndRelease (react-dom.development.js:3278)
    at executeDispatchesAndReleaseTopLevel (react-dom.development.js:3287)
    at forEachAccumulated (react-dom.development.js:3259)
    at runEventsInBatch (react-dom.development.js:3304)
    at runExtractedPluginEventsInBatch (react-dom.development.js:3514)
    at handleTopLevel (react-dom.development.js:3558)
    at batchedEventUpdates$1 (react-dom.development.js:21871)
    at batchedEventUpdates (react-dom.development.js:795)
    at dispatchEventForLegacyPluginEventSystem (react-dom.development.js:3568)
    at attemptToDispatchEvent (react-dom.development.js:4267)
    at dispatchEvent (react-dom.development.js:4189)
    at unstable_runWithPriority (scheduler.development.js:653)
    at runWithPriority$1 (react-dom.development.js:11039)
    at discreteUpdates$1 (react-dom.development.js:21887)
    at discreteUpdates (react-dom.development.js:806)
    at dispatchDiscreteEvent (react-dom.development.js:4168)
t @ oidc-client.min.js:47
onClick @ LoggedIn.tsx:12
callCallback @ react-dom.development.js:188
invokeGuardedCallbackDev @ react-dom.development.js:237
invokeGuardedCallback @ react-dom.development.js:292
invokeGuardedCallbackAndCatchFirstError @ react-dom.development.js:306
executeDispatch @ react-dom.development.js:389
executeDispatchesInOrder @ react-dom.development.js:414
executeDispatchesAndRelease @ react-dom.development.js:3278
executeDispatchesAndReleaseTopLevel @ react-dom.development.js:3287
forEachAccumulated @ react-dom.development.js:3259
runEventsInBatch @ react-dom.development.js:3304
runExtractedPluginEventsInBatch @ react-dom.development.js:3514
handleTopLevel @ react-dom.development.js:3558
batchedEventUpdates$1 @ react-dom.development.js:21871
batchedEventUpdates @ react-dom.development.js:795
dispatchEventForLegacyPluginEventSystem @ react-dom.development.js:3568
attemptToDispatchEvent @ react-dom.development.js:4267
dispatchEvent @ react-dom.development.js:4189
unstable_runWithPriority @ scheduler.development.js:653
runWithPriority$1 @ react-dom.development.js:11039
discreteUpdates$1 @ react-dom.development.js:21887
discreteUpdates @ react-dom.development.js:806
dispatchDiscreteEvent @ react-dom.development.js:4168
example-oidc-react:1 Unchecked runtime.lastError: The message port closed before a response was received.
----------------------------------------------------------------------------------------------------------------------------

When using npm package version 1.0.0-alpha.5 everything is Ok.

I've pulled version 1.0.0-alpha.5 (git checkout tags/v1.0.0-alpha.5  -b v1.0.0-alpha.5-branch)
Packed it tgz.  Everything is Ok.

So, version 1.0.0-alpha.5 was Ok, "develop" branch is not Ok.

**Comment by cobraz** [comment]
Hello!

Thanks for opening an issue. I haven't checked the error yet, but I wonder, what changes do you need?

**Comment by albertly** [comment]
@@ -59,6 +59,11 @@ export interface AuthProviderProps {
    */
   autoSignIn?: boolean;

+  /**
+   * defaults to { store: window.sessionStorage }
+   */
+  userStoreSettings?: WebStorageStateStoreSettings;
+
   /**
    * On before sign in hook. Can be use to store the current url for use after signing in.
    *

**Comment by cobraz** [comment]
Hmm, this could be something we’d like to add as to the library. Maybe you’d like to push a PR?

**Comment by albertly** [comment]
Yes, I'll be happy to. However I've made my changes on 1.0.0-alpha.5 (as working version from npm).
However, I have no possibility to check my changes on the latest one, because as I said it seems to be with error when using packed from another project.

When this issue be resolved, I'll make PR.

Best regard

**Comment by cobraz** [comment]
Hello 👋

I was not able to reproduce the issue you're having.

Maybe you can create a repro repository?
